docs/source/model_doc/mbart.rst

```diff
@@ -31,17 +31,9 @@ on the encoder, decoder, or reconstructing parts of the text.
 
 The Authors' code can be found `here <https://github.com/pytorch/fairseq/tree/master/examples/mbart>`__
 
-Examples
+Training of MBart
 _______________________________________________________________________________________________________________________
 
-- Examples and scripts for fine-tuning mBART and other models for sequence to sequence tasks can be found in
-  :prefix_link:`examples/seq2seq/ <examples/seq2seq/README.md>`.
-- Given the large embeddings table, mBART consumes a large amount of GPU RAM, especially for fine-tuning.
-  :class:`MarianMTModel` is usually a better choice for bilingual machine translation.
-
```

Comment:
Think this should a note in the examples or a forum post. I don't think we should compare different models in the docs.